src/routes/PublicRoute.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { element } from 'prop-types';
+
+export const PublicRoute = ({
+  component: Component,
+  ...rest
+}) => (
+  <Route
+    {...rest}
+    component={(props) => (
+      window.localStorage.jwtToken ? (
+        <>
+          <Component {...props} />
+        </>
+      ) : (
+        <>
+          <Component {...props} />
+        </>
+      )
```

Comment:
No need to check window.localStorage.jwtToken in public route

Reply:
I am planning to add NavBar to PublicRoute.